doc/misc/specs.7.rst

```diff
@@ -161,12 +161,12 @@ Specifying File Provides
 If a ``spec`` starts with either ``/`` or ``*/``, it is considered as a potential file provide.
 
 
-Groups
+Comps
 ======
 
-``<group-spec>`` allows one to select (environment) groups a particular operation should work
-on. It is a case insensitive string (supporting globbing characters) that is
-matched against a group's ID, canonical name and name translated into the
+``<group-spec>``|``<environment-spec>`` allows one to select groups|environments a particular
```

Comment:
Perhaps mention that commands accepting multiple types of specs uses the `@` prefix?

Reply:
Added.